pacta/README.md

```diff
@@ -1,91 +1,122 @@
+tag you provide will affect the image, the repository containing the
+dockerfile that builds the image, and the pacta siblings. That
+consistent tag make the process reproducible.
 
 To push the tags, start the container with something like
 
-```bash
+``` {.bash}
 cd ~
 docker run --rm -ti -v "$(pwd)/.ssh":/root/.ssh 2dii_pacta:latest
 ```
 
-This example starts an ephemeral container (`run --rm`) from the image `2dii_pacta:latest`, and creates a volume that makes your .ssh key available to the container (`-v "$(pwd)/.ssh":/bound/.ssh`), which you'll need to interact with GitHub. If instead of ssh you use https protocol, you may omit the volume argument and provide your username and password when prompted.
+This example starts an ephemeral container (`run --rm`) from the image
+`2dii_pacta:latest`, and creates a volume that makes your .ssh key
+available to the container (`-v "$(pwd)/.ssh":/bound/.ssh`), which
+you'll need to interact with GitHub. If instead of ssh you use https
+protocol, you may omit the volume argument and provide your username and
+password when prompted.
 
-Once inside the container, you can push any tag as you would normally push a branch `git push <remote> <tag>`, for example:
+Once inside the container, you can push any tag as you would normally
+push a branch `git push <remote> <tag>`, for example:
 
-```bash
+``` {.bash}
 cd /bound
 # If origin = https://github.com/2DegreesInvesting/PACTA_analysis
 git push origin 0.0.4
 ```
 
+You should also push the same tag that has been created in your local 
+2DegreesInvesting/docker repo.
+
 # For the web
 
 That shared docker image can be loaded into the new machine with...
 
-```docker load --input 2dii_pacta.tar.gz```
+`docker load --input 2dii_pacta.tar.gz`
 
 The docker image can then be used as intended with a script such as...
 
-```
-working_dir="$(pwd)"/working_dir
-user_results="$(pwd)"/user_results
-
-docker run --rm -ti \
-  --mount type=bind,source="$working_dir",target=/bound/working_dir \
-  --mount type=bind,source="$user_results",target=/user_results \
-  2dii_pacta \
-  /bound/bin/run-r-scripts
-```
-
-where you set `working_dir` to the path to the directory that contains the
-user specific portfolio info on the server, and you set `user_results` to
-the path to the directory that contains the survey (and other) results that
-are relevant to the specific user on the server. Those directories will then
-be mounted inside of the docker container in the appropriate locations.
+    portfolio_name="TestPortfolio"
+    working_dir="$(pwd)"/working_dir
+    user_results="$(pwd)"/user_results
+
+    docker run --rm -ti \
+      --network none \
+      --user 1000:1000 \
+      --memory="4g" \
+      --mount type=bind,source="$working_dir",target=/bound/working_dir \
+      --mount type=bind,source="$user_results",target=/user_results \
+      2dii_pacta \
+      /bound/bin/run-r-scripts "$portfolio_name"
```

Comment:
Nit: Maybe define this code chunk with backticks instead of space. Both produce the same but using backticks is consistent with the rest of the document.

Reply:
totally